language/move-vm/runtime/src/loader.rs

```diff
@@ -107,7 +105,7 @@ impl ScriptCache {
 // It holds all Modules, Types and Functions loaded.
 // Types and Functions are pushed globally to the ModuleCache.
 // A ModuleCache is accessed under lock.
-struct ModuleCache {
+pub(crate) struct ModuleCache {
```

Comment:
why is this going out? that sounds a tad scary....

Reply:
It doesn't have to. It's very likely I changed it for some reason that no longer exists. Will revert it before landing. Good catch BTW!